lib/app.js

```diff
@@ -1,31 +1,71 @@
+  
 const net = require('net');
 const parseRequest = require('./utils/parseRequest');
 const createResponse = require('./utils/createResponse');
 
-const app = net.createServer(client => {
+module.exports = net.createServer(client => {
   client.on('data', data => {
     const request = parseRequest(data.toString());
 
     // app.post('/red', () => ) for create
```

Comment:
Everything looked nice and clean, just make sure when submitting to get rid of these unnecessary console logs!